resources/windows/wix/service.mcollective.wxs.erb

```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+    <ComponentGroup Id="Service_MARIONETTECOLLECTIVE">
```

Comment:
Also We can't really make this a variable, since when vanagon would get to this erb it would not have context as to which service it was editing this file for, so unless we did something like take a hard line on the filename and say the filename needs to be the name of the service there is no way for us to know which service to put here.